uint32 command_num
---
CaptureRobotPixelsGoal config
float64[] command
